### ISSUE TITLE
Search bar disappear

### ISSUE THREAD

**Issue by Banh7** [ReVanced/revanced-manager#1900]
### Bug description

Search bar disappear only after few minutes when enable wide search bar in feed 
![Screenshot_2024-05-07-23-13-39-95_64ef5fc2000c1caa954c114bb372e1d5](https://github.com/ReVanced/revanced-manager/assets/94868053/a07f72e7-9f7c-4680-885c-ac4a6706ad5b)

### Version of ReVanced Manager and version & name of app you are patching

ReVanced Manager: 1.20.1
Youtube: 19.11.43

### Installation method

Regular

### ReVanced Manager logs

```shell
"alt_thumbnail_home": "dearrow",
"alt_thumbnail_library": "dearrow",
"alt_thumbnail_player": "dearrow",
"alt_thumbnail_search": "dearrow",
"alt_thumbnail_stills_time": "beginning",
"alt_thumbnail_subscription": "dearrow",
"announcement_last_id": 9,
"announcements": false,
"disable_resuming_shorts_player": true,
"external_downloader_action_button": true,
"hide_report_button": true,
"hide_seekbar_thumbnail": true,
"hide_shorts_channel_bar": true,
"hide_shorts_comments_button": true,
"hide_shorts_dislike_button": true,
"hide_shorts_full_video_link_label": true,
"hide_shorts_home": true,
"hide_shorts_like_button": true,
"hide_shorts_location_label": true,
"hide_shorts_paused_overlay_buttons": true,
"hide_shorts_save_sound_button": true,
"hide_shorts_search": true,
"hide_shorts_search_suggestions": true,
"hide_shorts_share_button": true,
"hide_shorts_sound_button": true,
"hide_shorts_sound_metadata_label": true,
"hide_shorts_subscriptions": true,
"hide_shorts_video_title": true,
"seekbar_custom_color": true,
"spoof_device_dimensions": true,
"swipe_lowest_value_enable_auto_brightness": true,
"tablet_layout": true,
"tablet_miniplayer": true,
"video_quality_default_wifi": 1080,
"wide_searchbar": true
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager
